### DISCUSSION TITLE
LaserGRBL

### DISCUSSION THREAD

**Discussion by MattTonning** [arkypita/LaserGRBL#1161] [Q&A]
I installed LaserGRBL the program opens but will not go any further.  I right-click import CustomButtons then the error comes back to say it cannot find CustomButton.GZ. What can I do next to make this software work?
![LaserGBL](https://user-images.githubusercontent.com/74028669/98449759-59260780-20eb-11eb-9bef-e8cdb8b8ba78.jpg)

**Comment by gmmanonymus111** [comment]
To import custom buttons, you first need to download a button package, then navigate to the download location (or navigate to where you exported your previous buttons). More [here](http://lasergrbl.com/usage/custom-buttons/)

To use LaserGRBL, you need to have a GRBL controlled laser engraver (I'm going to assume you have one). To connect to you engraver with a USB cable, first connect your machine to your PC (and also make sure you have installed the proper drivers for your engraver, CH340 one can be found in LaserGRBL: `Tools` -> `Install CH340 driver`)
![image](https://user-images.githubusercontent.com/9658243/98450105-26efc680-213a-11eb-8b12-2033b0075865.png)

- To connect, you need to find the correct COM port (if you have only one, most likely it is your machine), and select it at LaserGRBL's start screen. Then, set your BAUD rate, it is most likely `115200`, then click connect.
![image](https://user-images.githubusercontent.com/9658243/98450157-a2ea0e80-213a-11eb-994c-b241f5f87ee9.png)

- To connect wirelessly (for this, you of course need a wifi capabilities), go to `Grbl` -> `Settings` -> `Protocol` and set connection protocol to either `Telnet` or `LaserWeb` depending on your connection type.
![image](https://user-images.githubusercontent.com/9658243/98450240-784c8580-213b-11eb-83a8-46cba901f2ca.png)
Then, type the IP address of your machine (in case of `Telnet`, also use port):
![image](https://user-images.githubusercontent.com/9658243/98450258-92866380-213b-11eb-8a32-77c119eda83f.png)

Once done, click connect. (the plug icon near the IP address or the BAUD rate, depending which method you used)
A GRLB welcome message should appear:
![image](https://user-images.githubusercontent.com/9658243/98450450-24db3700-213d-11eb-8eb9-1cb91a23874f.png)

Also, it is advised to read [LaserGRBL's FAQ](http://lasergrbl.com/faq/)

**Comment by MattTonning** [comment]
Hi Again,

I am fully frustrated with this tool and the software. I have followed the
instructions you sent me but to no avail.
The software will not recognize the laser burner. Or even allow any manual
input. This is as far as I can get (see Pic attached)
Is it possible the software is not compatible with my windows 7 64? Since
it will not behave to your instructions?
I would love to get this thing working If not
[image: stumpted.jpg]
 I will send it back and look for a better product.

On Sat, Nov 7, 2020 at 12:08 PM Alex <notifications@github.com> wrote:

> To import custom buttons, you first need to download a button package,
> then navigate to the download location (or navigate to where you exported
> your previous buttons). More here
> <http://lasergrbl.com/usage/custom-buttons/>
>
> To use LaserGRBL, you need to have a GRBL controlled laser engraver (I'm
> going to assume you have one). To connect to you engraver with a USB cable,
> first connect your machine to your PC (and also make sure you have
> installed the proper drivers for your engraver, CH340 one can be found in
> LaserGRBL: Tools -> Install CH340 driver)
> [image: image]
> <https://user-images.githubusercontent.com/9658243/98450105-26efc680-213a-11eb-8b12-2033b0075865.png>
>
>    -
>
>    To connect, you need to find the correct COM port (if you have only
>    one, most likely it is your machine), and select it at LaserGRBL's start
>    screen. Then, set your BAUD rate, it is most likely 115200, then click
>    connect.
>    [image: image]
>    <https://user-images.githubusercontent.com/9658243/98450157-a2ea0e80-213a-11eb-994c-b241f5f87ee9.png>
>    -
>
>    To connect wirelessly (for this, you of course need a wifi
>    capabilities), go to Grbl -> Settings -> Protocol and set connection
>    protocol to either Telnet or LaserWeb depending on your connection
>    type.
>    [image: image]
>    <https://user-images.githubusercontent.com/9658243/98450240-784c8580-213b-11eb-83a8-46cba901f2ca.png>
>    Then, type the IP address of your machine (in case of Telnet, also use
>    port):
>    [image: image]
>    <https://user-images.githubusercontent.com/9658243/98450258-92866380-213b-11eb-8a32-77c119eda83f.png>
>
> Once done, click connect. (the plug icon near the IP address or the BAUD
> rate, depending which method you used)
> A GRLB welcome message should appear:
> [image: image]
> <https://user-images.githubusercontent.com/9658243/98450450-24db3700-213d-11eb-8eb9-1cb91a23874f.png>
>
> Also, it is advised to read LaserGRBL's FAQ <http://lasergrbl.com/faq/>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/arkypita/LaserGRBL/issues/1161#issuecomment-723489021>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARUZM7JMH6NS2PRF2IX7GWLSOWSK3ANCNFSM4TN2TR7A>
> .
>


-- 

*Matt P Tonning III*
*www.YPGuy.co <http://www.WestPage.com>m*
*(360) 464-8347 *

Truth will always be the truth, regardless of lack of understanding,
disbelief or ignorance.

*W Clement Stone*
